newsfragments/9180.feature.rst

```diff
@@ -0,0 +1 @@
+[CLI] Implement invite claim and greet for shared recovery.
```

Comment:
Those newsfragments are meant to be user oriented. I think we should have only one of them for the whole #6090 meta-issue, introducing the shared recovery feature in general